validation/js/src/main/scala/hmda/validation/rules/ts/syntactical/S028.scala

```diff
@@ -0,0 +1,15 @@
+package hmda.validation.rules.ts.syntactical
+
+import hmda.model.fi.ts.TransmittalSheet
+import hmda.validation.dsl.{ Result, HmdaDsl, CommonDsl }
```

Comment:
(This is now moot, but I'm adding the comment just so we/I have a record of it.)  I played a bit with ways to remove the duplication, which naturally all boil down to passing some kind of dependency into the rule.  I didn't find a way to do it and preserve the nice DSL-ness of the syntax, but it was still educational (with regard to both scala and this system).  And it might not be the last time we encounter a question of running things differently in different contexts.  It does seem like traits mean you have to decide certain things at compile time.